repose-aggregator/components/filters/compression-filter/src/main/scala/org/openrepose/filters/compression/CompressionFilter.scala

```diff
@@ -91,6 +93,7 @@ class CompressionFilter @Inject()(configurationService: ConfigurationService, co
           logger.error("Error with the CompressingFilter", e)
           httpResponse.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR)
       }
+      httpResponse.commitToResponse()
```

Comment:
Yes. It failed the unit test, but passed the functional test. This change made it pass the unit test also.